app/assets/components/lks-header/_header.scss

```diff
@@ -72,6 +228,10 @@
   margin: 0;
   padding-left: 0;
 
+  &.multiLogo {
```

Comment:
Could we follow BEM naming convention for `multiLogo` ?

Reply:
Done